<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsssWindow</class>
 <widget class="QWidget" name="OsssWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 18pt &quot;华文楷体&quot; ;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1280</width>
     <height>760</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
/*background-color: #e0afce91;*/
background-color: rgba(175, 206, 145,100);

border: 10px solid rgb(200, 235, 170);
border-radius: 20px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="text">
     <string>刘述凯-学号230207163019</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>730</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="text">
     <string>230207163019学号-刘述凯</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>1227</x>
      <y>13</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
    /* 基础样式 - 未选中状态（红色） */
	font: 20pt &quot;等线&quot;;
	border-radius: 20px;
	   background-color: #FF5252;  /* 红色 */

}

QPushButton:hover  {
    /* 选中状态（浅红色） */
background-color: #FFCDD2;  /* 浅红色 */
 
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>270</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_2{
    border-radius: 20px;
    
    /* 主边框 */
    border: 1px solid #9cba7f;
    
    /* 模拟内高光 - 使用内边距和背景渐变 */
    padding: 2px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 rgba(255,255,255,0.2), 
                              stop:0.5 rgba(255,255,255,0.05),
                              stop:1 rgba(0,0,0,0.05));
    
    /* 立体边框效果 */
    border-top: 3px solid rgba(230, 242, 215, 0.9);
    border-left: 3px solid rgba(230, 242, 215, 0.9);
    border-bottom: 4px solid #7a9a62;
    border-right: 4px solid #7a9a62;
    
    /* 模拟外阴影 - 使用外边距和轮廓 */
    margin: 2px;
    outline: 1px solid rgba(0,0,0,0.1);
	background-color: #E8F5E9;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>展示基本业务：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_tips_Price">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>收货地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
     </property>
     <property name="text">
      <string>触发器1: 无法下单已经停止合作或停止生产的产品</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>数：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_employee">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>对接人</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cb_customer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>购买方</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="sb_quantity">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stoping">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>290</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_tips_Price_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>价格</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>下单：(存储过程)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_product">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>购买商品</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>270</width>
      <height>310</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_3{
border-radius: 20px;
    
    /* 主边框 */
    border: 1px solid #9cba7f;
    
    /* 模拟内高光 - 使用内边距和背景渐变 */
    padding: 2px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 rgba(255,255,255,0.2), 
                              stop:0.5 rgba(255,255,255,0.05),
                              stop:1 rgba(0,0,0,0.05));
    
    /* 立体边框效果 */
    border-top: 3px solid rgba(230, 242, 215, 0.9);
    border-left: 3px solid rgba(230, 242, 215, 0.9);
    border-bottom: 4px solid #7a9a62;
    border-right: 4px solid #7a9a62;
    
    /* 模拟外阴影 - 使用外边距和轮廓 */
    margin: 2px;
    outline: 1px solid rgba(0,0,0,0.1);
background-color: #F1F8E9;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_savefun_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>活跃供应商(存储函数)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>价格变动：(存储过程)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_update">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sb_newPrice">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_savefun_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>库存总价值(存储函数)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>新价格：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_reason">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>原因</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_product_update">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>货物</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>920</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_4{
border-radius: 20px;
    
    /* 主边框 */
    border: 1px solid #9cba7f;
    
    /* 模拟内高光 - 使用内边距和背景渐变 */
    padding: 2px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 rgba(255,255,255,0.2), 
                              stop:0.5 rgba(255,255,255,0.05),
                              stop:1 rgba(0,0,0,0.05));
    
    /* 立体边框效果 */
    border-top: 3px solid rgba(230, 242, 215, 0.9);
    border-left: 3px solid rgba(230, 242, 215, 0.9);
    border-bottom: 4px solid #7a9a62;
    border-right: 4px solid #7a9a62;
    
    /* 模拟外阴影 - 使用外边距和轮廓 */
    margin: 2px;
    outline: 1px solid rgba(0,0,0,0.1);
background-color: #DCEDC8 ;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>480</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>触发器2: 订单状态变化为'已完成'时删减库存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_state">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>460</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>890</width>
       <height>360</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;华文楷体&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_order_state">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>460</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>未选择</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_del_order">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>460</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>订单管理状态管理：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_select_state">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>460</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>已完成</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已发货</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>待处理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已取消</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>选择订单：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>460</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>选择更改状态：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>订单状态：</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>920</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_5{
border-radius: 20px;
    
    /* 主边框 */
    border: 1px solid #9cba7f;
    
    /* 模拟内高光 - 使用内边距和背景渐变 */
    padding: 2px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 rgba(255,255,255,0.2), 
                              stop:0.5 rgba(255,255,255,0.05),
                              stop:1 rgba(0,0,0,0.05));
    
    /* 立体边框效果 */
    border-top: 3px solid rgba(230, 242, 215, 0.9);
    border-left: 3px solid rgba(230, 242, 215, 0.9);
    border-bottom: 4px solid #7a9a62;
    border-right: 4px solid #7a9a62;
    
    /* 模拟外阴影 - 使用外边距和轮廓 */
    margin: 2px;
    outline: 1px solid rgba(0,0,0,0.1);
background-color: #EaFbE0  ;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_show_order_detial">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>客户订单明细(视图)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_show_sales_summary">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>110</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>销售统计视图(视图)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_10">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_9">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_show_inventory_overview">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>60</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>库存概览视图(视图)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_8">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_6">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_query_5">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>10</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查询3</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
